src/main/java/de/retest/recheck/ignore/SearchFilterFiles.java

```diff
@@ -56,4 +58,21 @@ private SearchFilterFiles() {}
 			return Collections.emptyList();
 		}
 	}
+
+	public static Optional<Filter> searchFilterByName( final String name ) throws IOException {
+		for ( int j = 0; j < getAllFilterFiles().size(); j++ ) {
```

Comment:
You are calling `getAllFilterFiles()` multiple times. Please extract a variable.